content/en/docs/collector/management.md

```diff
@@ -0,0 +1,229 @@
+   backpressure-related information.
+1. Connection management between a control plane and the agent such as handling
+   of TLS certificates (revocation and rotation).
+
+Not every use case requires support for all of the above agent management tasks.
+In the context of OpenTelemetry task _4. Health and performance monitoring_ is
+ideally done using OpenTelemetry.
+
+## OpAMP
+
+Observability vendors and cloud providers offer proprietary solutions for agent
+management. In the open source observability space, there is an emerging
+standard that you can use for agent management: Open Agent Management Protocol
+(OpAMP).
+
+The [OpAMP specification][opamp-spec] defines how to manage a fleet of telemetry
+data agents. These agents can be [OpenTelemetry collectors][otel-collector],
+Fluent Bit or other agents in any arbitrary combination.
+
+> **Note** The term "agent" is used here as a catch-all term for OpenTelemetry
+> components that respond to OpAMP, this could be the collector but also SDK
+> components.
+
+OpAMP is a client/server protocol that supports communication over HTTP and over
+WebSockets:
+
+- The **OpAMP server** is part of the control plane and acts as the
+  orchestrator, managing a fleet of telemetry agents.
+- The **OpAMP client** is part of the data plane. The client side of OpAMP can
+  be implemented in-process, for example, as the case in [OpAMP support in the
+  OpenTelemetry collector][opamp-in-otel-collector]. The client side of OpAMP
+  could alternatively be implemented out-of-process. For this latter option, you
+  can use a supervisor that takes care of the OpAMP specific communication with
+  the OpAMP server and at the same time controls the telemetry agent, for
+  example to apply a configuration or to upgrade it. Note that the
+  supervisor/telemetry communication is not part of OpAMP.
+
+Let's have a look at a concrete setup:
+
+![OpAMP example setup](../img/opamp.svg)
+
+1. The OpenTelemetry collector, configured with pipeline(s) to:
+   - (A) receive signals from downstream sources
+   - (B) export signals to upstream destinations, potentially including
+     telemetry about the collector itself (represented by the OpAMP `own_xxx`
+     connection settings).
+1. The bi-directional OpAMP control flow between the control plane implementing
+   the server-side OpAMP part and the collector (or a supervisor controlling the
+   collector) implementing OpAMP client-side.
+
+You can try out a simple OpAMP setup yourself by using the [OpAMP protocol
+implementation in Go][opamp-go]. For the following walkthrough you will need to
+have Go in version 1.19 or above available.
+
+We will set up a simple OpAMP control plane consisting of an example OpAMP
+server and let an OpenTelemetry collector connect to it via an example OpAMP
+supervisor.
+
+First, clone the `open-telemetry/opamp-go` repo:
+
+```sh
+$ git clone https://github.com/open-telemetry/opamp-go.git
```

Comment:
When there's no command output, and the code-block language is `sh` (or `shell` or `bash`), drop the prompt:

```suggestion
git clone https://github.com/open-telemetry/opamp-go.git
```